go.mod

```diff
@@ -253,9 +250,9 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101411.5-rc.1
+replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101411.5-rc.1.0.20241217162941-94bb3f660f77
```

Comment:
we need to reference the rc tag once the op-geth PR is merged

Reply:
done